[2024-08-31T17:52:14.872+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-31T17:52:15.156+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [queued]>
[2024-08-31T17:52:15.169+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [queued]>
[2024-08-31T17:52:15.170+0000] {taskinstance.py:2306} INFO - Starting attempt 21 of 22
[2024-08-31T17:52:15.194+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): execution_time> on 2024-08-31 16:50:00.890695+00:00
[2024-08-31T17:52:15.204+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3301) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-31T17:52:15.206+0000] {standard_task_runner.py:64} INFO - Started process 3303 to run task
[2024-08-31T17:52:15.205+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'execution_time', 'manual__2024-08-31T16:50:00.890695+00:00', '--job-id', '351', '--raw', '--subdir', 'DAGS_FOLDER/envioemail.py', '--cfg-path', '/tmp/tmpk92s5ut7']
[2024-08-31T17:52:15.208+0000] {standard_task_runner.py:91} INFO - Job 351: Subtask execution_time
[2024-08-31T17:52:15.269+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [running]> on host 19bb8daff211
[2024-08-31T17:52:15.408+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='jorgepons90@gmail.com,facturasjorgepons@gmail.com' AIRFLOW_CTX_DAG_OWNER='Jorge Pons' AIRFLOW_CTX_DAG_ID='simple_dag' AIRFLOW_CTX_TASK_ID='execution_time' AIRFLOW_CTX_EXECUTION_DATE='2024-08-31T16:50:00.890695+00:00' AIRFLOW_CTX_TRY_NUMBER='21' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-31T16:50:00.890695+00:00'
[2024-08-31T17:52:15.411+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-31T17:52:15.443+0000] {logging_mixin.py:188} INFO - <class 'datetime.timedelta'>
[2024-08-31T17:52:15.444+0000] {logging_mixin.py:188} INFO - 0:00:00.286502
[2024-08-31T17:52:15.476+0000] {base.py:84} INFO - Using connection ID 'my_mssql_conn_id' for task execution.
[2024-08-31T17:52:15.528+0000] {logging_mixin.py:188} INFO - 1
[2024-08-31T17:52:15.531+0000] {logging_mixin.py:188} INFO - Error: Unsupported parameter type: <class 'datetime.timedelta'>
[2024-08-31T17:52:15.535+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-31T17:52:15.537+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-31T17:52:15.568+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=execution_time, run_id=manual__2024-08-31T16:50:00.890695+00:00, execution_date=20240831T165000, start_date=20240831T175215, end_date=20240831T175215
[2024-08-31T17:52:15.624+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-31T17:52:15.654+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-31T17:52:15.658+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
